tests/dummy/app/styles/app.scss

```diff
@@ -1,3 +1,6 @@
+// TODO This should be imported somewhere in the addon
+@import "ember-power-select";
```

Comment:
Is there somewhere else (in the addon) I can add this?

Reply:
can you not add them to `addon/styles/components/dropdown.scss`?

Reply:
When I put it there, the build fails and says it's unable to find the stylesheet